docs/ECLStandardLibraryReference/SLR-Mods/SuperFileExists.xml

```diff
@@ -40,12 +40,14 @@
   TRUE if the specified <emphasis>filename</emphasis> is present in the
   Distributed File Utility (DFU) and is a SuperFile. It returns FALSE if the
   <emphasis>filename</emphasis> does exist in the DFU but is not a SuperFile
-  (i.e. is a normal DATASET—use the STD.File.FileExists function to detect
+  (i.e., is a normal DATASET—use the STD.File.FileExists function to detect
```

Comment:
Was that new comma intentional? Could also do with rewording/simplifying this sentence I think.
```
(Use Std.File.FileExists to check presence of a standard file)
```
..perhaps..

>..does exist in the DFU

would read better as:

.. does exist in DFU